client.ts

```diff
@@ -24,6 +25,21 @@ export class Client {
     return await this._connection.query(query);
   }
 
+  async transaction(
```

Comment:
This method doesn't really have anything in common with `transaction`... Maybe just `multiQuery`?

Reply:
sure, fixed now